### DISCUSSION TITLE
ONNX model fails to load in browser

### DISCUSSION THREAD

**Discussion by elaith9** [snakers4/silero-vad#97] [Q&A] [answered]
Hello I'm trying to load onnx model using JS in browser. I'm using official example from ONNX github:

```
<html>
  <head> </head>

  <body>
    <!-- Load ONNX.js -->
    <script src="https://cdn.jsdelivr.net/npm/onnxjs/dist/onnx.min.js"></script>
    <!-- Code that consume ONNX.js -->
    <script>
      // create a session
      const myOnnxSession = new onnx.InferenceSession();
      // load the ONNX model file
      myOnnxSession.loadModel("./my-model.onnx").then(() => {
        // generate model input
        const inferenceInputs = getInputs();
        // execute the model
        myOnnxSession.run(inferenceInputs).then((output) => {
          // consume the output
          const outputTensor = output.values().next().value;
          console.log(`model output tensor: ${outputTensor.data}.`);
        });
      });
    </script>
  </body>
</html>

```
But loading fails with following message. Do you have any idea what might cause this? I found no information on onnx website.
![image](https://user-images.githubusercontent.com/17851129/131363723-585d47d2-bde5-4297-872c-357e8751cef6.png)

**Comment by snakers4** [comment]
To be honest, we have not tried these models on JS, but this model was reported to work on mobile successfully - https://github.com/snakers4/silero-vad/blob/master/files/model_micro_mobile.jit

Last time someone from community tried ONNX.js it was abandoned and kind of did not work

Their README points to this page now - https://github.com/microsoft/onnxruntime/tree/master/js/web

**Comment by snakers4** [comment]
Same issue here is stale - https://github.com/microsoft/onnxjs/issues/302

**Comment by elaith9** [comment]
> To be honest, we have not tried these models on JS, but this model was reported to work on mobile successfully - https://github.com/snakers4/silero-vad/blob/master/files/model_micro_mobile.jit
> 
> Last time someone from community tried ONNX.js it was abandoned and kind of did not work
> 
> Their README points to this page now - https://github.com/microsoft/onnxruntime/tree/master/js/web

Is there any example of running .jit model on Android? Thanks

**Comment by snakers4** [comment]
> Is there any example of running .jit model on Android? Thanks

Some people who requested the `model_micro_mobile.jit` promised to publish some form of guide
But they did not `(ノಠ益ಠ)ノ彡┻━┻`

**Comment by snakers4** [comment, marked as answer]
Running silero-vad on Android - https://github.com/bgubanov/VadExample
You need to use this model - https://github.com/snakers4/silero-vad/blob/master/files/model_micro_mobile.jit

**Comment by snakers4** [comment]
New ONNX model released - https://github.com/snakers4/silero-vad/releases/tag/v3.1
Also ONNX recently got another new backend to run in JS
